<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceSettingsDialog</class>
 <widget class="QDialog" name="ServiceSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="serviceFullNameLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Service full name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serviceFullNameEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="serviceFilenameLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Service filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serviceFilenameEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="serviceLaunchPathLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Service launch path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serviceLaunchPathEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="serviceLaunchPathButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="serviceLaunchParamsLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Service Launch Params</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="serviceLaunchParamsEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="serviceLaunchCommandLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Service Launcher Command</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serviceLaunchCommandEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="serviceParamsLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Service params</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="serviceParamsEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="serviceTypeLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Service Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serviceTypeBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="serviceLaunchDelayLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Service Launch Delay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="serviceLaunchDelayBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="startedRegexLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Started regex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="startedRegexEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="resultLaunchCommandLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Result Launch Command</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="resultLaunchCommandEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceFilenameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>paramChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceFilenameEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>paramChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceFilenameEdit</sender>
   <signal>cursorPositionChanged(int,int)</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>paramChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceLaunchParamsEdit</sender>
   <signal>textChanged()</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>paramChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceLaunchParamsEdit</sender>
   <signal>cursorPositionChanged()</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>paramChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceLaunchCommandEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>paramChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceLaunchCommandEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>paramChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceLaunchCommandEdit</sender>
   <signal>cursorPositionChanged(int,int)</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>paramChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceParamsEdit</sender>
   <signal>textChanged()</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>paramChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceParamsEdit</sender>
   <signal>cursorPositionChanged()</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>paramChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceTypeBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ServiceSettingsDialog</receiver>
   <slot>paramChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>paramChanged()</slot>
 </slots>
</ui>
